### DISCUSSION TITLE
Azure Github Intergration.

### DISCUSSION THREAD

**Discussion by PaarlBoysHigh** [community/community#57522] [GitHub Education]
### Hi everyone,

I need to Setup Github to intergrate with Azure forour school. Can anyone help

**Comment by chriskthomas** [comment]
So you mean that you want SSO with Active Directory? You need GitHub Campus for that.

**Comment by PaarlBoysHigh** [comment]
Yes. Ok let me check out community

**Reply by mwt** [reply]
https://education.github.com/schools
